src/applications/accredited-representative-portal/tests/e2e/accredited-representative-portal.cypress.spec.js

```diff
@@ -10,32 +10,21 @@ const vamcUser = {
   },
 };
 
-const arpUserLOA3 = {
-  ...user,
-  login: {
-    currentlyLoggedIn: true,
-  },
-};
-
 Cypress.Commands.add('loginArpUser', () => {
   cy.intercept('GET', '**/accredited_representative_portal/v0/user', {
     statusCode: 200,
-    body: arpUserLOA3,
+    body: user,
   }).as('fetchUser');
 });
 
 const setUpInterceptsAndVisit = featureToggles => {
   cy.intercept('GET', '/data/cms/vamc-ehr.json', vamcUser).as('vamcUser');
-  cy.intercept('GET', 'http://localhost:3000/v0/user', {
-    statusCode: 200,
-    body: {},
-  }).as('getUser');
   setFeatureToggles(featureToggles);
   cy.visit('/representative');
   cy.injectAxe();
 };
 
-describe('Accredited Representative Portal', () => {
+describe.skip('Accredited Representative Portal', () => {
```

Comment:
This test has stale 21-22 code that needs to be cleaned up.

We'll fix this `.skip` as a part of the below code cleanup issue:
https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/92083